sdk/storage/azure-storage-file-share/pom.xml

```diff
@@ -33,6 +33,16 @@
   </scm>
 
   <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core</artifactId>
+      <version>1.14.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.9.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core-http-netty;dependency} -->
+    </dependency>
```

Comment:
Why did we have to explicitly add these dependencies? Storage common should bring in the unreleased version of core and netty right?

Reply:
This prevent transitive resolution errors where the additional hop results in the dependencies resolving to an unexpected version of these libraries.